pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -160,10 +160,18 @@ func Apply(clientSet clientset.Interface, target *deployments.Target) error {
 	if err != nil {
 		return err
 	}
+
 	// bsc#1155810: generate cluster-wide kubelet root certificate, and generate/rotate kuberlet server certificate
-	if err := target.Apply(nil, "kubelet.rootca.create", "kubelet.servercert.create"); err != nil {
+	if err := kubernetes.GenerateKubeletRootCert(); err != nil {
 		return err
 	}
+	err = target.Apply(nil,
+		"kubelet.rootcert.upload",
+		"kubelet.servercert.create.upload")
```

Comment:
It would be good to merge this one with the next list, `target.Apply` will run them in order.

Reply:
Resolved.